scripts/ci_locally.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+set -e
+set +x
+
+
+echo -e '\n\n# Run CI jobs locally'
+echo 'Disclaimer: local environment may differ from Github Actions environment.'
+
+
+echo -e '\n\n### Step: Setup Node'
+echo 'Disclaimer: CI runs matrix on v20.17.0 and 22. Please eyeball if your local version matches.'
+node -v
+echo 'Disclaimer: CI caches node_modules, skipping locally.'
+
+echo -e '\n\n### Step: Install JavaScript dependencies (npm install)'
+npm install
+
+echo -e '\n\n### Step: Check Biome formatting, import order and linter'
+npm run lint:biome
+
+echo -e '\n\n### Step: Check TypeScript'
+npm run lint:types
+
+echo -e '\n\n### Step: Build Prod'
+SKIP_TS_CHECK=true npm run build
+
+echo -e '\n\n### Step: Check ESLint, errors only'
+npm run lint:eslint -- --quiet
+
+echo -e '\n\n### Step: Run unit tests and xlform tests'
+npx jest --config ./jsapp/jest/unit.config.ts --ci
+
+echo -e '\n\n### Step: Run component tests with Jest'
+npx jest --config ./jsapp/jest/jest.config.ts --ci
+
+
+echo -e '\n\n## Job: Darker'
+
+echo -e '\n\n### Step: Set up Python'
+echo 'Disclaimer: CI installs Python 3.10. Please eyeball if your local env has 3.10.'
+python --version
+
+echo -e '\n\n### Step: Install pip dependencies'
+python -m pip install darker[isort] flake8 flake8-quotes isort --quiet
```

Comment:
Is this a fail-fast optimization for CI? If all of `dev_requirements.txt` ends up being installed anyway, maybe we could avoid and just run `darker` later?

Reply:
As per discussed, I removed all python dependencies installation. Everything should be already installed in the container of a dev environment.